### PR TITLE
fix: use `SMISMEMBER` if multiple keys are given

### PR DESCRIPTION
**Pull request by insertish**
Ran into an issue trying to use this when I found out it isn't implemented.
I basically copied how `GET`/`MGET` works, hopefully that's fine.

If there's anything wrong here I'll be happy to ammend the PR.

### CODE REVIEW

**Comment by jaymell** [comment]
Hi, thanks for the PR. What you did is totally reasonable, however I'd actually prefer it if we just added a separate command. Do you mind adding a separate command for `SMISMEMBER` instead?

**Comment by insertish** [comment]
Done

**Comment by jaymell** [comment]
Can we add a simple test for it as well?

File: redis/src/commands/mod.rs [hunk 1]
```diff
@@ -512,6 +512,11 @@ implement_commands! {
         cmd("SISMEMBER").arg(key).arg(member)
     }
 
+    /// Determine if multiple values are members of a set.
+    fn smismember<K: ToRedisArgs, M: ToRedisArgs>(key: K, member: M) {
```
Comment (shachlanAmazon) [inline redis/src/commands/mod.rs:516]: minor: please rename member to members, to clarify that the function can take multiple values there.

**Comment by nihohit** [comment]
@Zomatree why did you add changes to the pub/sub to this PR?

**Comment by Zomatree** [comment]
My mistake I was doing other changes, I didn't release this pr was set for this branch.

**Comment by nihohit** [comment]
@Zomatree @insertish 
sorry, somehow missed your comment.
Please fix the comment about the argument name and add a test, and then let's merge this :)

**Comment by nihohit** [comment]
implemented in https://github.com/redis-rs/redis-rs/pull/1002

**Pull request closed without merging**